src/main/java/com/frcteam3636/frc2024/subsystems/shooter/Shooter.kt

```diff
@@ -153,26 +148,31 @@ object Shooter {
         }
         private val inputs = PivotIO.Inputs()
         private var pivotOffset: Double = 0.0
-        private val leftLimitSwitchUnpressed = DigitalInput(2)
-        private val rightLimitSwitchUnpressed = DigitalInput(3)
 
         override fun periodic() {
-//            if(!leftLimitSwitchUnpressed.get() || !rightLimitSwitchUnpressed.get()){
-//                pivotOffset = -inputs.position.radians
-//            }
             io.updateInputs(inputs)
             Logger.processInputs("Shooter/Pivot", inputs)
 
             armLigament.angle = inputs.position.degrees
+
             Logger.recordOutput("Shooter", mechanism)
         }
 
         fun pivotAndStop(goal: Rotation2d): Command = Commands.sequence(runOnce {
+            Logger.recordOutput("Shooter/DesiredPosition", goal)
             io.pivotToAndStop(goal)
-        }, Commands.waitUntil {
-            (abs((goal - inputs.position).radians) < PIVOT_POSITION_TOLERANCE.radians)
-                    && (abs(inputs.velocity.radians) < PIVOT_VELOCITY_TOLERANCE.radians)
         })
+//            , Commands.waitUntil {
+//            (abs((goal - inputs.position).radians) < PIVOT_POSITION_TOLERANCE.radians)
+//                    && (abs(inputs.velocity.radians) < PIVOT_VELOCITY_TOLERANCE.radians)
+//        })
+
+
```

Comment:
Might be good to look over whether this should be commented out/removed

Reply:
it def should not be commented out or removed, its super necessary once we get better system integration

Reply:
Okay, I think we should get these changes into main very soon since they have important fixes for the drivetrain. So I might uncomment these for now and make a separate branch for kG testing where they can be disabled for now.